tests/VampIR/positive/Compilation/data/test022.json

```diff
@@ -1,5 +1,5 @@
 {
     "in1": "5",
-    "in2": "11",
-    "out": "48828125"
+    "in2": "9",
+    "out": "1953125"
 }
```

Comment:
There is no newline at the end of the file

Reply:
That's how VSCode JSON formatter formats JSON files

Reply:
It is a convention that all text files should be ended by a newline. Even github ui warns you about it. There must be a way to configure vscode to change that behaviour.

some refs:
1. https://saturncloud.io/blog/why-should-text-files-end-with-a-newline/
2. https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file
3. https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
Okay, I know this convention, but let's do it in a separate PR as it would change all JSON files we have.

Reply:
* #2349